bin/templates/project/Activity.kt

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
If we make Kotlin an optional feature, this file should not be used and defaults to `Activity.java`. Only use this file if the EnableKotlin flag is set to `true`.

Reply:
This file is actually not unnecessary.. I'm taking it out.